src/main/java/xyz/deseteral/lebkuchenfm/services/commands/CommandProcessor.java

```diff
@@ -0,0 +1,26 @@
+package xyz.deseteral.lebkuchenfm.services.commands;
+
+import xyz.deseteral.lebkuchenfm.domain.Command;
+import xyz.deseteral.lebkuchenfm.domain.CommandProcessingResponse;
+
+import java.util.List;
+import java.util.Optional;
+
+interface CommandProcessor {
```

Comment:
Da się podzielić ten interfejs na 4 mniejsze? Będą łatwiejsze w utrzymaniu i testowaniu.

Reply:
No właśnie - nie wiem. Chciałem to zrobić, ale nie wydaje mi żebym mógł. 

Problem polega na tym, że klasy które są "komendami" muszą implementować ten interfejs, bo w `CommandExecutor` wstrzykuje sobie `List<CommandProcessor>`. W ten sposób mam listę wszystkich "komend" jakie aplikacja ma zaimplementowane. 

Z drugiej strony w interfejsie `CommandProcessor` mam minimalny kontrakt jaki musi spełniać komenda.